src/game/logic/ai/aiplayer.cpp

```diff
@@ -0,0 +1,518 @@
+/**
+ * @file
+ *
+ * @author Jonathan Wilson
+ *
+ * @brief AI Player
+ *
+ * @copyright Thyme is free software: you can redistribute it and/or
+ *            modify it under the terms of the GNU General Public License
+ *            as published by the Free Software Foundation, either version
+ *            2 of the License, or (at your option) any later version.
+ *            A full copy of the GNU General Public License can be found in
+ *            LICENSE
+ */
+#include "aiplayer.h"
+#include "ai.h"
+#include "gamelogic.h"
+#include "player.h"
+#include "scriptengine.h"
+#ifdef GAME_DLL
+#include "hooker.h"
+#endif
+
+void Delete_Queue(TeamInQueue *team)
+{
+    if (team != nullptr) {
+        team->Delete_Instance();
+    }
+}
+
+AIPlayer::AIPlayer(Player *p) :
+    m_player(p),
+    m_readyToBuildTeam(false),
+    m_readyToBuildStructure(false),
+    m_teamTimer(2),
+    m_structureTimer(2),
+    m_teamDelaySeconds(10),
+    m_buildDelay(0),
+    m_teamDelay(0),
+    m_skillsetSelector(INVALID_SKILLSET_SELECTION),
+    m_dozerID(OBJECT_UNK),
+    m_structureRepairCount(0),
+    m_dozerQueued(false),
+    m_supplySourceFrame(0),
```

Comment:
m_dozerRepairing
m_bridgeRepairTimer
m_supplySourceFrame

not initialized

Reply:
fixed.